apps/fz_http/lib/fz_http/oidc/start_proxy.ex

```diff
@@ -4,37 +4,41 @@ defmodule FzHttp.OIDC.StartProxy do
   (after `FzHttp.Configurations.Cache` has started) and pass to `OpenIDConnect.Worker`
   """
 
-  alias FzHttp.Configurations, as: Conf
-
   require Logger
+  import Actual.Cache
 
   def child_spec(arg) do
     %{id: __MODULE__, start: {__MODULE__, :start_link, [arg]}}
   end
 
   def start_link(:test) do
-    auth_oidc_env = Conf.get!(:openid_connect_providers)
-    Conf.Cache.put!(:parsed_openid_connect_providers, parse(auth_oidc_env))
+    auth_oidc_env = cache().get!(:openid_connect_providers)
+    cache().put!(:parsed_openid_connect_providers, parse(auth_oidc_env))
```

Comment:
So many changes related to the cache, PR would look cleaner if we just drop it right away, but I guess it's not that important now.

Reply:
That's funny. I had exactly the same thought. I'll take a stab this morning. It should be quick.